### ISSUE TITLE
allow combining fonts

### ISSUE THREAD

**Issue by KiaraGrouwstra** [danth/stylix#276]
css unit `unicode-range` allows [specifying fonts by character range](https://stackoverflow.com/questions/11395584/fallback-fonts-on-special-characters).
it would be nice if stylix were to facilitate such a use-case, so as to e.g. allow specifying different fonts for different languages.

**Comment by danth** [comment]
Possibly related to #201

**Comment by danth** [comment]
Yes, what you're requesting will require more than what the current PR provides.

It may already be possible to build you're requesting on top of Stylix by merging the fonts into a single font (using `pkgs.runCommand`), giving that font a different name, and passing that package and name into the Stylix options.
